x-pack/plugins/fleet/common/index.ts

```diff
@@ -64,6 +64,7 @@ export {
 } from './services';
 
 export type { FleetAuthz } from './authz';
+// eslint-disable-next-line @kbn/imports/no_boundary_crossing
 export { createFleetAuthzMock } from './mocks';
```

Comment:
why does this line need a disable comment?

Reply:
Mocks should not be exported from common/browser/server production code:

> tests or mocks: code that is loaded by jest/storybook and mocks/helpers intended for use by that code. These files usually live along side package code but will have a separate dependency tree and are pieces of code which should never end up in the product.

Reply:
I see, thanks for clarifying.